ansible/README.md

```diff
@@ -43,5 +43,15 @@ to enter the SSH password:
 The `--tags` option tells Ansible to only execute tasks with the specified tag.
 In the previous examples, only tasks with the `setup` tag would be executed.
 
-Provide the data requested by the playbook (eg Jenkins admin user name/password
-and SSH keys locations) and wait automatic setup finish.
+For Jenkins master playbook you only need to provide a jenkins username and password in 
+the target host.
```

Comment:
Not true, there are four prompts in the master playbook: https://github.com/open-power-host-os/infrastructure/blob/master/ansible/jenkins-master.yaml#L21

Reply:
Oops, my mistake.